cypress/e2e/irctc.cy.js

```diff
@@ -38,7 +38,8 @@ describe('IRCTC TATKAL BOOKING', () => {
       // TATKAL or NORMAL BOOKING
       if (TATKAL) {
         cy.get('#journeyQuota > .ui-dropdown').click()
-        cy.get(':nth-child(6) > .ui-dropdown-item').click()
+        //cy.get(':nth-child(6) > .ui-dropdown-item').click() //For Tatkal
+        cy.get(':nth-child(7) > .ui-dropdown-item').click() //FOr Premium Tatkal
```

Comment:
Premium / Normal Tatkal can be driven by a config rather than hardcoding premium.